src/main/java/io/github/queerbric/inspecio/Inspecio.java

```diff
@@ -141,9 +141,9 @@ static String getVersion() {
 	 * @param tooltip the tooltip
 	 */
 	public static void appendBlockItemTooltip(ItemStack stack, Block block, List<Text> tooltip) {
-		var config = Inspecio.get().getConfig().getContainersConfig().forBlock(block);
+		StorageContainerConfig config = Inspecio.get().getConfig().getContainersConfig().forBlock(block);
 		if (config != null && config.hasLootTable()) {
-			var blockEntityNbt = stack.getSubNbt(BlockItem.BLOCK_ENTITY_TAG_KEY);
+			NbtCompound blockEntityNbt = stack.getSubNbt("BlockEntityTag");
```

Comment:
I'm not very happy about that, it works but ideally it should not hardcode the value.